timeseries/src/autogluon/timeseries/models/presets.py

```diff
@@ -247,6 +249,7 @@ def get_preset_models(
                 prediction_length=prediction_length,
                 eval_metric=eval_metric,
                 eval_metric_seasonal_period=eval_metric_seasonal_period,
+                metadata=metadata,
```

Comment:
This change is just making a `kwarg` explicit in the signature of `get_preset_models`. This one was committed by mistake, but I think we can keep since it has no effect on the code.